examples/libp2p-in-the-browser/package.json

```
@@ -9,7 +9,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "parcel index.html",
-    "server": "star-signal"
+    "webrtc-star": "star-signal",
```

This is another option for running the `webrtc-star` server and it will also match the logging mentioned in the readme.
```suggestion
    "webrtc-star": "webrtc-star",
```